charmhelpers/contrib/openstack/utils.py

```diff
@@ -1841,3 +1841,30 @@ def series_upgrade_complete(resume_unit_helper=None, configs=None):
         configs.write_all()
         if resume_unit_helper:
             resume_unit_helper(configs)
+
+
+def run_in_apache(package, release=None):
```

Comment:
I don't think this function is particularly useful in charm-helpers.  Basically, it's being specialised for each of the charms by package.  These will be simpler, smaller, functions, if they are just put in each charms' utils.py or equivalent file.

I'd say it will impede understanding in a charm if this function is in charm-helpers, and for each new charm that needs this functionality, charm-helpers will need to be modified.  It's not a common function.  Please remove it.